projects/packages/waf/tests/php/integration/test-waf-activation.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Activation tests.
+ *
+ * @package automattic/jetpack-waf
+ */
+
+use Automattic\Jetpack\Waf\Waf_Initializer;
+use Automattic\Jetpack\Waf\Waf_Runner;
+
+/**
+ * Activation tests.
+ */
+final class WafActivationTest extends WorDBless\BaseTestCase {
+
+	/**
+	 * Test setup.
+	 *
+	 * @before
```

Comment:
I gather from the previous conversation here that we don't actually need the `@before` on this side due to the polyfill from the `WorDBless\BaseTestCase`? Unless I just understood that wrong it might be better to remove this here, it probably doesn't hurt in this instance either, though.

Reply:
Ah, I misunderstood - removed the `@before` tag here: https://github.com/Automattic/jetpack/pull/28980/commits/5b1acad446ac5a30d14347561f2db80cbd3b699f

Thanks for catching that 👍 👍

Reply:
Whoops - discovered while writing additional tests that the `@before` tag is necessary 😅

Looks like `WorDBless\BaseTestCase` extends `PHPUnit\Framework\TestCase`, and doesn't have its own implementation of `set_up`.

I reverted the commit.

Reply:
oops, ok :)

Reply:
Are you sure about that? https://github.com/Automattic/wordbless/blob/84d9fe02cc3b222d03dd62ad3145a5265beccb0b/src/BaseTestCase.php#L5-L7 shows it using the Yoast base class.

That hasn't been included in a release version yet, but your composer.json is specifying `@dev` so it should be getting the linked version.

Reply:
Interesting, we do have the `@dev` version specified but it does look like the latest `0.4.1` release gets downloaded by composer into `jetpack-waf/vendor/automattic/wordbless`.